CONTRIBUTION.md

```diff
@@ -27,8 +27,28 @@ We are very happy that you consider implementing algorithms and data structure f
 - If you have modified/added code work, make sure the code compiles before submitting.
 - **Be consistent in use of there guidelines when submitting**
 
+#### New File Name guidelines
+- Use lowercase words with ``"_"`` as separator
+- For instance 
+```
+MyNewCppClass.CPP       is incorrect
+my_new_cpp_class.cpp    is correct format
+```
+- It will be used to dynamically create a directory of files and implementation.
+- File name validation will run on docker to ensure the validity.
+
+#### New Directory guidelines 
+- Use lowercase words with ``"_"`` as separator ( no spaces or '-' allowed )
```

Comment:
Do we want to say that it is best practice to try to use an existing directory rather than adding a new one?

Reply:
added chagne